src/java.desktop/share/classes/javax/imageio/metadata/doc-files/tiff_metadata.html

```diff
@@ -574,7 +574,8 @@ <h3><a id="Compression">Compression</a></h3>
 <th scope="row">9</th>
 <td>Exif JPEG</td>
 <td>Exif-specific JPEG compression (see note following this table)</td>
-<td><a href="http://www.exif.org/Exif2-2.PDF">Exif 2.2 Specification</a>
+<td><a href="https://www.kodak.com/global/plugins/acrobat/en/service/digCam/exifStandard2.pdf">
+    Exif 2.2 Specification</a>
```

Comment:
This doesn't seem like a very authorative URL.
It looks quite unstable and could be gone at the whim of whoever maintains that website describing their acrobat plugins ..

Reply:
I found another link `http://web.archive.org/web/20131018091152/http://exif.org/Exif2-2.PDF` for spec version 2.2 which is a web archive but the connection is not secure.

Is it ok to have `https://www.cipa.jp/std/documents/e/DC-008-2012_E.pdf` link and update the Spec version to 2.3 ?

What do you suggest?

Reply:
I found the webarchive myself yesterday, I did not think it a suitable link for JDK.
Since you haven't found a suitable link for 2.2, I think an update to 2.3 is best mainly because it seems a better place to link to.

Reply:
Updated to version 2.3.

Reply:
Can we place that document somewhere(I think that is not easy but still) and provide a link to it?

Reply:
Did you mean this docment `https://www.kodak.com/global/plugins/acrobat/en/service/digCam/exifStandard2.pdf` ?

I am not sure if we can do this.

Reply:
I mean the Exif2-2.PDF from the "web.archive"

Reply:
I don't have an idea where we can put `Exif2-2.PDF` and provide a link. 
Since we have a suitable link to version 2.3, it's better to point to that link. As mentioned by @prrace also in the comment above. 
https://github.com/openjdk/jdk20/pull/74#discussion_r1056618094.